style-guide.md

```diff
@@ -0,0 +1,311 @@
+
+**Shell Code Samples**
+
+Start shell code samples with `~~~ shell` followed by a line break. The first character of the next line must be the terminal marker `$`.
+
+**SQL Code Samples**
+
+SQL code samples are broken into two sections: commands and responses.
+
+- **Commands** (e.g., `SELECT`, `CREATE TABLE`) should begin with `~~~ sql` followed by a line break. The first character of the next line must be the terminal marker `>`. Commands should be properly capitalized, and there should be only one command per code sample.
+- **Responses** (e.g., retrieved tables) should begin with `~~~` but should *not* be syntax highlighted.
+
+  Note that not all responses warrant inclusion. For example, if a SQL code sample shows `CREATE TABLE`, `INSERT`, and then `SELECT`, it's unnecessary to show the responses for `CREATE TABLE` (which is just `CREATE TABLE`) and `INSERT` (which is just `INSERT <number of rows>`).
+
+**Copy to Clipboard**
+
+Many of our code blocks are written so users can copy and paste them directly into a terminal. To make that easier, add a **Copy to Clipboard** feature by placing
+
+<pre>
+{% raw %}{<span>% include copy-clipboard.html %}{% endraw %}
+</pre>
+
+on the line directly preceding the code block, for example:
+
+<pre>
+{% raw %}{% include copy-clipboard.html %}{% endraw %}
+~~~ shell
+$ go get -u github.com/lib/pq
+~~~
+</pre>
+
+**Copy to Clipboard** should be used for every code block that can be executed.
+
+### Examples
+
+"How to title them
+How to introduce them
+Grammatical person to use (you vs we)
+Split sentence by code or syntax?"
+
+### Headings
+
+Use headings to demarcate content into a hierarchy to help readers find information easier.
+
+Headings should be title case, and are denoted by octothorps (`#`) followed by one space to denote headings. Enter a line break between a heading and its content. CockroachDB docs use Heading 2 (`##`) and Heading 3 (`###`). Heading 1 is reserved for page titles and anything under Heading 3 can be denoted by bolded text, or should be reformatted.
+
+Examples:
+
+- `## This is Heading 2`
+- `### And This is Heading 3`
+
+### Images
+
+Use images to clarify a topic, and should be used only as needed. Images are either:
+
+- **Screenshots** - Provide a UI visual. Screenshots should show enough of the UI that the user can easily orient themselves and understand what they are being shown. If a screenshot needs an annotation, use a red box.
+- **Diagrams** - Provide a visual of a complicated theory. Diagrams should be simple and easy to read.
+
+Use the following HTML and liquid to include an image in a markdown page:
+
+~~~ html
+<img src="{% raw %}{{ 'images/v2.1/image-name.png' | relative_url }}{% endraw %}" alt="Alternative Text Here" style="border:1px solid #eee;max-width:100%" />
```

Comment:
We need to use {% raw %}? God, Lauren, I am so glad you took up this project. I am learning so much!!